apps/prairielearn/src/pages/instructorCourseAdminStaff/coursePermissionsInsertForm.ejs

```diff
@@ -9,8 +9,9 @@
   </div>
 
   <div class="form-group">
-    <label for="addUsersInputUid">List of UIDs separated by commas, whitespaces, or semicolons:</label>
-    <textarea class="form-control" id="addUsersInputUid" name="uid" placeholder="user1@domain.org, user2@domain.org" required></textarea>
+    <label for="addUsersInputUid">UIDs:</label>
+    <textarea class="form-control" id="addUsersInputUid" name="uid" placeholder="user1@domain.org, user2@domain.org" required aria-describedby="addUsersInputUidHelp"></textarea>
```

Comment:
Change `domain.org`?

Reply:
Done in 43ee1fa; there were a few other places throughout the codebase that I changed as well.

Reply:
Excellent, thanks.